test/auth/auth.e2e-spec.ts

```diff
@@ -51,21 +51,29 @@ describe('AuthController (e2e)', () => {
       isAccountDisabled: false,
       email: 'e2etester@random.com',
     };
+
     const registerOutput: RegisterOutput = {
       id: 2,
       name: 'e2etester',
       username: 'e2etester',
       roles: [ROLE.USER],
       isAccountDisabled: false,
       email: 'e2etester@random.com',
+      createdAt: null,
```

Comment:
Does it return `null` when you do POST request?
Can you check if we can return the new values that are set on the DB by doing a `refresh: true` flag if necessary. :thinking:

Reply:
@yashmurty 
> Does it return null when you do POST request?
No

I do this in following lines 
```
          registerOutput.createdAt = resp.data.createdAt
          registerOutput.updatedAt = resp.data.updatedAt
```